Overview.md

```diff
@@ -0,0 +1,65 @@
+
+# Confidential Containers
+We are interested in integrating existing [Trusted Execution Environments](https://en.wikipedia.org/wiki/Trusted_execution_environment) (TEE) 
+infrastructure support and technologies with the cloud native world. Our focus is to place a kubernetes pod into a TEE. 
```

Comment:
Are Confidential Containers inherently connected to pod-centric virtualization? Could we support other runtimes that isolate one container at a time?